v1/src/main/java/com/google/cloud/teleport/spanner/ExportPipeline.java

```diff
@@ -182,6 +182,8 @@ public interface ExportPipelineOptions extends PipelineOptions {
     @TemplateParameter.Text(
         order = 10,
         optional = true,
+        parentName = "shouldExportRelatedTables",
+        parentTriggerValues = {"true"},
```

Comment:
I think this needs to be reverted.  I don't know this template well, but the `shouldExportRelatedTables` parameter description says "If `tableNames` is specified but doesn't include related tables, this option must be set to true for a successful export." Which means in some cases `tableNames` parameter can be set to something with `shouldExportRelatedTables`  set to false. So we should show `tableNames` unconditionally.